server/shared/src/main/scala/org/http4s/server/staticcontent/FileService.scala

```diff
@@ -62,6 +63,7 @@ object FileService {
     /** For binary compatibility.
       * @return an instance of PathCollector[F] created (converted) from pathCollector2
       */
+    @nowarn
```

Comment:
Should this be deprecated?

```suggestion
    @deprecated
```

Reply:
yes, that's the one I missed